cmd/kube-apiserver/app/options/features.go

```diff
@@ -0,0 +1,18 @@
+package options
+
+import (
+	genericfeatures "k8s.io/apiserver/pkg/features"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/kubernetes/pkg/features"
+)
+
+var featureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
+	features.ExternalTrafficLocalOnly: {Default: true, PreRelease: utilfeature.GA},
```

Comment:
In terms of ensuring components set the right values (`Default` and `PreRelease` are the only ones I assume), perhaps we can centralise the `FeatureSpec` structs in `pkg/feature` and then have the commands assemble the lists themselves.